### DISCUSSION TITLE
Should preview change the current region?

### DISCUSSION THREAD

**Discussion by kenkangxgwe** [minad/consult#1027] [Ideas]
When a mark is activated and `consult-line` is called, previewing the candidate will extend the current region to the candidate's point. Is this an intended behavior? If not, shall we `deactivate-mark ` before previewing and maybe re-activate it if `consult-line` is canceled?

**Comment by minad** [comment]
I think either way, there are advantages and disadvantages.

**Comment by kenkangxgwe** [comment]
Got it. Thanks for the claim.
